services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -75,7 +80,7 @@ const StateUserRoutes = ({
 }): React.ReactElement => {
     // feature flag
     const ldClient = useLDClient()
-    const showRateSummaryPage: boolean = ldClient?.variation(
+    const showRateEditPage: boolean = ldClient?.variation(
```

Comment:
Nit: for semantics, if this case is now covering showing rate summary and edit pages it might be clearer to name it something that covers both, like `showRatePages`